benches/files.rs

```diff
@@ -68,20 +68,23 @@ fn benchmark_files_no_partitions(c: &mut Criterion) {
                     options.create(true).write(true);
 
                     let input = options.open(input).await.unwrap();
-                    let input = FramedWrite::new(input, BytesCodec::new())
-                        .sink_map_err(|e| panic!("{:?}", e));
 
                     (topology, input)
                 });
                 (rt, topology, input)
             },
             |(rt, topology, input)| {
                 rt.block_on(async move {
-                    let lines = random_lines(line_size).take(num_lines).map(|mut line| {
+                    let mut sink = FramedWrite::new(input, BytesCodec::new());
+                    //.sink_map_err(|e| panic!("{:?}", e));
```

Comment:
Left some commented-out code here.

Reply:
😓 

I'll make a note to clean that up.  Thanks for spotting it.

Reply:
I'll be fixing this in https://github.com/vectordotdev/vector/pull/12572, FWIW, since I'm already working on that branch.